### PR TITLE
Nc 18

### PR DESCRIPTION
**Pull request by ccellist**
Updated Dockerfile to new version of Nextcloud, version 18. Dockerfile has been tested and is running in production at https://bach.arayahome.net.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -118,7 +118,7 @@ COPY root /
 
 RUN chmod +x /usr/local/bin/run.sh /usr/local/bin/occ /etc/periodic/15min/nextcloud
 
-VOLUME ["/data","/opt/nextcloud"]
+VOLUME ["/data"]
```
Comment (ccellist) [inline Dockerfile]: I had made this change locally at one point for testing but rolled it back to what you have for consistency. In other words, this line matches master.

**Comment by burner** [comment]
@chrootlogin @ccellist any progress on this?

**Comment by ccellist** [comment]
Unfortunately I don't have access to merging pull requests on this project. I've tested this PR and it's been working great for me. Hoping @chrootlogin can provide guidance.

**Comment by ccellist** [comment]
This PR may not be necessary. Reading the readme, since the version is an ARG all you have to do is pass --build-arg NEXTCLOUD_VERSION=18.0.3 (for current version) and you'll get the version you need. It may be necessary to include NEXTCLOUD_GPG as well, if that ever changes). I may close this if this doesn't get accepted soon.

**Comment by chrootlogin** [comment]
Thank you! :+1:

**Comment by burner** [comment]
thank you

**Pull request merged**